tests/scsi/007

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright 2021 Google LLC
+#
+# Trigger the SCSI error handler.
+
+. tests/scsi/rc
+. common/scsi_debug
+
+DESCRIPTION="Trigger the SCSI error handler"
+
+QUICK=1
+
+requires() {
+	_have_scsi_debug
+}
+
+# Override the function with the same name defined in tests/scsi/rc such that
+# this test is run whether or not the test device is a SCSI device.
+group_device_requires() {
+	true
+}
```

Comment:
This test defines the `test()` function, not the `test_device()` function. That means that it is independent of any test devices, so this is unnecessary (see `tests/scsi/004`, which is similar).